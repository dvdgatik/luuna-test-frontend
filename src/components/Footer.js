import React from 'react';

function Footer() {
	return(
		<footer>
			<p className='footer-title'>Created by David Gatica | 2021</p>
		</footer>
	)

}



export default Footer;