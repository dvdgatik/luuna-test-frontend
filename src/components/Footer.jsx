import React from 'react';
import styled from 'styled-components';

const StyledFooter = styled.footer`
	background:#002355;
	padding:20px;
	color: #ffffff;

`;
const Paragraph = styled.p`   
`;

const  Footer = props => {
	return(
		<StyledFooter>
			<Paragraph className='footer-title'>Created by David Gatica | 2021</Paragraph>
		</StyledFooter>
		
	);

}



export default Footer;