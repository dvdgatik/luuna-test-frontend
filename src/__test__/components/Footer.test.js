import React from 'react';
import Footer from '../../components/Footer';
import {create} from 'react-test-renderer';
import {mount, shallow} from 'enzyme';
import styled from 'styled-components'
import 'jest-styled-components';


describe('<Footer>',()=>{
	const footer = mount(<Footer/>);
	test('Render of Footer Component', () => {
		expect(footer.length).toEqual(1);
	});
	test('Render of title', ()=> {
		expect(footer.find(".footer-title > p").text()).toEqual('Created by David Gatica | 2021');
	});
});


describe('Footer Snapshot', () => {
	test('Check UI of Footer Component',()=>{
		const footer = create(<Footer/>);
		expect(footer.toJSON()).toMatchSnapshot();
	})
});