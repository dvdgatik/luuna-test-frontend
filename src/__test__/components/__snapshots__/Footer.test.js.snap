// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Snapshot Check UI of Footer Component 1`] = `
.c0 {
  background: #002355;
  padding: 20px;
  color: #ffffff;
}

<footer
  className="c0"
>
  <p
    className="footer-title"
  >
    Created by David Gatica | 2021
  </p>
</footer>
`;
